src/index.ts

```diff
@@ -18,27 +36,7 @@ type RequiredModelSubset = Pick<
 	'tables' | 'relationships' | 'synonyms'
 >;
 
-const typeHelpers = {
-	read: `
-export type DateString = string;
-export type Expanded<T> = Extract<T, any[]>;
-export type PickExpanded<T, K extends keyof T = keyof T> = {
-	[P in K]-?: Expanded<T[P]>;
-};
-export type Deferred<T> = Exclude<T, any[]>;
-export type PickDeferred<T, K extends keyof T = keyof T> = {
-	[P in K]: Deferred<T[P]>;
-};
-export interface WebResource {
-	filename: string;
-	href: string;
-	content_type?: string;
-	content_disposition?: string;
-	size?: number;
-};
-`,
-	write: '',
-};
+const typeHelpers = `import type { Types } from '@balena/abstract-sql-to-typescript';\n`;
```

Comment:
Could this come from `package.json`  instead of hardcoding the package name?

Reply:
I think I slightly prefer hardcoded in this specific case so we don't have to load package.json at runtime and deal with the whole including it in the build (it's a bit of a pain). But it did make me realize that it would actually make sense to split this file into two; the runtime part (ie exporting helper types) and the compile time part (ie generating types) so that you don't have to import both at runtime

Reply:
Separated them